packages/app-extensions/src/actions/modules/actionHandlers/legacyAction.js

```diff
@@ -48,6 +49,11 @@ export default function* (definition, selection, parent, params, config) {
 
   const actionDefinition = new window.nice2.netui.actions.model.ClientActionDefinition(definition.id, definition.id)
   actionDefinition.setEnabled(!definition.readOnly)
+  if (_isPlainObject(definition.properties)) {
+    Object.keys(definition.properties).forEach(propertyName => {
+      actionDefinition.setProperty(propertyName, definition.properties[propertyName])
```

Comment:
if you want to test your code later you probably have to rewrite it to `yield call(actionDefintion.setProperty, propertyName....`